packages/@aws-cdk/aws-cloudfront/README.md

```diff
@@ -225,6 +225,25 @@ myDistribution.addBehavior('images/*', myOrigin, {
 });
 ```
 
+Adding an existing Lambda@Edge function created in a different stack to a CloudFront distribution.
+
+```ts
+const functionVersion = lambda.Version.fromVersionArn(this, 'Version', 'arn:aws:lambda:us-east-1:123456789012:function:functionName:1');
+
+const distro = new cloudfront.Distribution(this, "distro", {
+  defaultBehavior: {
+    origin: new origins.S3Origin(s3Bucket),
+    edgeLambdas: [
+       {
+         functionVersion,
+         eventType: cloudfront.LambdaEdgeEventType.VIEWER_REQUEST
+       },
+    ],
+  },
+  ...
```

Comment:
I'd rather just remove this, as the code is perfectly valid without, and not having uncompilable bits will help with us translating this across languages.

```suggestion
```